src/components/menu/Menu.tsx

```diff
@@ -247,6 +247,14 @@ const Menu = <S,>({
     }
   }, [calculatePosition, positionState, maxHeight]);
 
+  useEffect(() => {
+    if (active) {
+      document.body.style.overflow = 'hidden';
+    } else {
+      document.body.style.overflow = '';
```

Comment:
I think you have to do something like https://github.com/teamleadercrm/ui/blob/next-release/src/components/overlay/Overlay.tsx#L63-L73 to prevent incorrectly unlocking the scroll

Reply:
@lorgan3 Isn't this the same? Do you mean the `lockscreen` prop?